src/ledger/src/multisig.rs

```diff
@@ -109,42 +109,40 @@ fn submit_send(
         timeout,
         execute_automatically,
     } = multisig_arg;
-    let symbol = crate::resolve_symbol(&client, symbol)?;
-
-    if client.id.identity.is_anonymous() {
-        Err(ManyError::invalid_identity())
-    } else {
```

Comment:
That reminds me that we should add `#[deny_anonymous]` to additional methods in `many-rs`.

https://github.com/liftedinit/many-rs/issues/197